src/util/helpers.ts

```diff
@@ -13,6 +13,9 @@ export function defaultGetNextRetryDelay(retryNumber: number): number {
 
   return Math.min(backoff, MAX_BACKOFF) + randomMs;
 }
+/**
```

Comment:
```suggestion

/**
```

for consistency with other docstrings, there should be one empty line before